config/install/core.date_format.alert_date.yml

```
@@ -1,7 +1,7 @@
 langcode: en
 status: true
 dependencies: {  }
-id: govcms_alert_date
+id: alert_date
```

```suggestion
id: alert_banner_date
```

All references of `govcms_alert` must be changed to `alert_banner`, not `alert`.

---

@sonnykt Updated